### PR TITLE
(MAINT) Write 2020-11-20 Status Update

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

**Comment by DavidS** [comment]
cc @logicminds @nwops - anything else you want to add to the gitpod paragraph?

File: _posts/2020-11-20-status-update.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  - team
+  - status
+tags:
+---
+
+Coming at you with the latest weekly blog post, here to fill you in on all the juicy IAC gossip!
+
+## Gitpod
+
+As of a few days ago a pr ([pdk-templates#354][pdk-templates-354]), submitted by the execellent [logicminds][logicminds] has been merged in, adding support for gitpod puppet development to the pdk-templates as an opt in option, allowing contributors to edit any enabled module via VSCode from the browser without requiring any local setup or install.
```
Comment (DavidS) [inline _posts/2020-11-20-status-update.md:15]:
```suggestion
As of a few days ago a pr ([pdk-templates#354][pdk-templates-354]), submitted by the excellent [logicminds][logicminds] has been merged in, adding support for gitpod puppet development to the pdk-templates as an opt in option, allowing contributors to edit any enabled module via VSCode from the browser without requiring any local setup or install.
```

File: _posts/2020-11-20-status-update.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+If you'd like to see the rest of our planned work for the next cycle of releases, you can peek in on our [active milestone on GitHub][dsc-milestone] and generally watch our [GitHub project][dsc-project] for all the work in flight!
+
+## Cloud CI Update
+
+This week we managed to port the puppetlabs-apache and puppetlabs-java modules to run using our new Cloud CI setup.
+We’ve prepared the templates in [pdk-templates](pdk-templates) for the nightly and PR Github Action workflows that will be used in the future for porting the rest of the modules.
+
+The puppetlabs-iis porting PR is still waiting to be merged, however we have stumbled upon some acceptance test issues there that we are currently investigating.
+
+For next week we will be focusing on improving the provision service and the associated task error handling and stabilization improvement, ironing out any remaining bumps we stumble across as we port more and more modules to use the new CI solution.
```
Comment (DavidS) [inline _posts/2020-11-20-status-update.md:47]:
```suggestion
For next week we will be focusing on improving the provision service and the associated task error handling and stabilization improvement, ironing out any remaining bumps we stumble across as we port more and more modules to use the new CI solution.

Some required changes for deploying the Github Actions to all our modules (and the reporting from the changes) can be seen in [puppetlabs/pdk-templates#363](https://github.com/puppetlabs/pdk-templates/pull/363)
```

**Review by DavidS — APPROVED** [review]
two minor changes proposed

File: _posts/2020-11-20-status-update.md (hunk 1 above)
Comment (david22swan) [reply _posts/2020-11-20-status-update.md]: Done

File: _posts/2020-11-20-status-update.md (hunk 2 above)
Comment (david22swan) [reply _posts/2020-11-20-status-update.md:47]: Done

File: _posts/2020-11-20-status-update.md (hunk 1 above)
Comment (logicminds) [reply _posts/2020-11-20-status-update.md]: Maybe say that,  a container is creating the gitpod environment that comes packed with pdk, puppet-debugger, puppet-agent, zsh, oh-my-zsh plugins and other productivity improvements to enhance puppet development.

**Review by daianamezdrea — APPROVED** [review]

**Pull request merged**